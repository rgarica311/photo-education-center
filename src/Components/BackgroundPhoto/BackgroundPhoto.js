import React, { Component } from 'react';

const bgPhotoStyle = {
  width: '100vw',
  height: '100vh',
  backgroundSize: 'cover',
  backgroundImage: `url('http://creativephotoacademy.com/wp-content/uploads/2014/01/aKenya2011-5016.jpg')`,
  backgroundRepeat: 'no-repeat'
};

export default class BackgroundPhoto extends Component {
  constructor( props ) {
    super( props );

  };

  render() {
    return (
      <div style={bgPhotoStyle} className='bg-photo'></div>
    );
  }

}
