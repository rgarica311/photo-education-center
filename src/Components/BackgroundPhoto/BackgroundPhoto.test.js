import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import BackgroundPhoto from './BackgroundPhoto';

describe(`BackgroundPhoto Component`, () => {

  it('renders BackgroundPhoto by default', () => {
    const wrapper = shallow(<BackgroundPhoto />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
