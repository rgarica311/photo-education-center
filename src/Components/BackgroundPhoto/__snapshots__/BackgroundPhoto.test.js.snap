// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundPhoto Component renders BackgroundPhoto by default 1`] = `
<div
  className="bg-photo"
  style={
    Object {
      "backgroundImage": "url('http://creativephotoacademy.com/wp-content/uploads/2014/01/aKenya2011-5016.jpg')",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "cover",
      "height": "100vh",
      "width": "100vw",
    }
  }
/>
`;
